demo/src/javascripts/components/demos/arc.js

```diff
@@ -108,16 +111,14 @@ export default class ArcDemo extends Component {
     selectedFeature = selectedFeature || features[362];
 
     const {flows, centroid, name} = selectedFeature.properties;
-    const arcs = [];
-
-    for (let toId in flows) {
+    const arcs = Object.keys(flows).reduce((acc, toId) => {
       const f = features[toId];
-      arcs.push({
+      return [...acc, {
```

Comment:
Using `[...array]` in `reduce` has a big perf hit (`O(N^2)`). 
Babel transforms the following:
```
arr.reduce((acc, d) => [...acc, d], []);
```
To:
```
function _toConsumableArray(arr) { if (Array.isArray(arr)) { for (var i = 0, arr2 = Array(arr.length); i < arr.length; i++) { arr2[i] = arr[i]; } return arr2; } else { return Array.from(arr); } }

arr.reduce(function (acc, d) {
  return [].concat(_toConsumableArray(acc), [d]);
}, []);
```